backend/src/api/routers/health.py

```
@@ -0,0 +1,15 @@
+from fastapi import APIRouter, Request
+from fastapi.responses import JSONResponse
+
+router = APIRouter(prefix="/health", tags=["health"])
+
+@router.get("/", response_class=JSONResponse)
+async def health_check():
+    return JSONResponse(content={"status": "ok"})
+    
+@router.get("/module", response_class=JSONResponse)
+async def health_check():
```

Ruff (0.14.10)

Redefinition of unused `health_check` from line 7

(F811)